<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomPublishDate_w</class>
 <widget class="QDialog" name="CustomPublishDate_w">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>501</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ps:发布时间不得是现在以前的时间，否则可能被视为历史记录;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>若同一个终端发布时间相同将被视为更新操作</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="year_l">
      <property name="text">
       <string>年</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="year_e"/>
    </item>
    <item>
     <widget class="QLabel" name="mon_l">
      <property name="text">
       <string>月</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="mon_e"/>
    </item>
    <item>
     <widget class="QLabel" name="day_l">
      <property name="text">
       <string>日</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="day_e"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>195</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="ok_b">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
